spellcheck: локализация инженерных инструментов

Dimempers opened the pull request:
Основной перевод всех инженерных инструментов вместе с суицидами и сообщениями.

<!-- Пишите **НИЖЕ** заголовков и **ВЫШЕ** комментариев, иначе ваш текст может не отобразиться. -->
<!-- В Contributing.MD вы можете найти некоторые рекомендации к оформлению пулл-реквеста. -->

## Описание
Основной перевод всех инженерных инструментов вместе с суицидами и сообщениями в чате.
<!-- Опишите, что делает ваш Pull request. Документировать каждую деталь не требуется, просто укажите основные изменения. -->

## Причина создания ПР / Почему это хорошо для игры
Локализация игры
<!-- Здесь можно оставить ссылку на сообщение в #отчеты-по-предложениям, чтобы подтвердить, что предложение обсуждалось внутри Discord-сообщества. -->
<!-- Если отчёта нет, то укажите, почему это изменение положительно влияет на игру. -->
<!-- В случае исправления бага, укажите ссылку на канал в #баг-репорты-v2 или issue в репозитории. В ином случае, опишите баг и ступени для его воспроизведения. -->
<!-- Пример ссылки в Discord-сообщество : https://discord.com/channels/617003227182792704/755125334097133628/ID-сообщения -->

## Демонстрация изменений

<!-- В случае наличия изменений, влияющих на игровую часть, опишите их здесь. В случае их отсутствия, этот пункт можно удалить -->

## Тесты
Зашел на локалку, осмотрел перебрал все инструменты, все на русском, все чики пуки.
<!-- Здесь необходимо описать шаги, которые предпринимались для тестирования изменения. Этот пункт обязателен, без него Pull request будет рассматриваться дольше. -->

Dimempers committed:
Translation-engi-tools

Основной перевод всех инженерных инструментов вместе с суицидами и сообщениями.

Dimempers committed:
Поправил суицид отверткой

Dimempers committed:
Merge branch 'master220' into Translation-of-engineering-tools

PlayerUnknown14 commented on code/game/objects/items/tools/crowbar.dm:
```diff
@@ -1,6 +1,6 @@
 /obj/item/crowbar
 	name = "pocket crowbar"
-	desc = "A small crowbar. This handy tool is useful for lots of things, such as prying floor tiles or opening unpowered doors."
+	desc = "Небольшой лом. Этот удобный инструмент полезен для многих вещей, таких как поддевание напольной плитки или открытие дверей без электропитания."
```
`"дверей без электропитания" - "обесточенных шлюзов"`

PlayerUnknown14 commented on code/game/objects/items/tools/crowbar.dm:
```diff
@@ -17,7 +17,7 @@
 	attack_verb = list("attacked", "bashed", "battered", "bludgeoned", "whacked")
 	toolspeed = 1
-
+	ru_names = list(NOMINATIVE = "корманный лом", GENITIVE = "корманного лома", DATIVE = "корманному лому", ACCUSATIVE = "корманный лом", INSTRUMENTAL = "корманным ломом", PREPOSITIONAL = "корманном ломе")
```
* Принято писать в таком виде, см. скриншот.
![image](https://github.com/user-attachments/assets/393de64f-5f19-47df-8d14-0682dba7e69c)


кАрманный*

Перемести выше под `desc`.

PlayerUnknown14 commented on code/game/objects/items/tools/crowbar.dm:
```diff
@@ -36,26 +36,28 @@
 	icon_state = "crowbar_brass"
 	item_state = "crowbar_brass"
 	toolspeed = 0.5
+	ru_names = list(NOMINATIVE = "латунный лом", GENITIVE = "латунного лома", DATIVE = "латунному лому", ACCUSATIVE = "латунный лом", INSTRUMENTAL = "латунным ломом", PREPOSITIONAL = "латунном ломе")
```
Также.

PlayerUnknown14 commented on code/game/objects/items/tools/crowbar.dm:
```diff
@@ -36,26 +36,28 @@
 /obj/item/crowbar/brass
 	name = "brass crowbar"
-	desc = "A brass crowbar. It feels faintly warm to the touch."
+	desc = "Латунный лом. На ощупь он слегка теплый."
```
* Пишем "ё", не заменяем на "е".


`"На ощупь он слегка теплый" - "Слегка тёплый на ощупь"`

PlayerUnknown14 commented on code/game/objects/items/tools/crowbar.dm:
```diff
@@ -36,26 +36,28 @@
 	item_state = "alien_crowbar"
 	belt_icon = "alien_crowbar"
 	toolspeed = 0.1
+	ru_names = list(NOMINATIVE = "инопланетный лом", GENITIVE = "инопланетного лома", DATIVE  = "инопланетному лому", ACCUSATIVE = "инопланетный лом", INSTRUMENTAL = "инопланетным ломом", PREPOSITIONAL = "инопланетном ломе")
```
Также.

PlayerUnknown14 commented on code/game/objects/items/tools/crowbar.dm:
```diff
@@ -36,26 +36,28 @@
 /obj/item/crowbar/large
 	name = "crowbar"
-	desc = "It's a big crowbar. It doesn't fit in your pockets, because its too big."
+	desc = "Это большой лом. Он не поместится в кармане, потому что он слишком большой."
```
`"Он не поместится в кармане, потому что он слишком большой" - "Такой в карман не положить."`

PlayerUnknown14 commented on code/game/objects/items/tools/crowbar.dm:
```diff
@@ -64,17 +66,19 @@
 	icon_state = "crowbar_large"
 	item_state = "crowbar_large"
 	toolspeed = 0.5
+	ru_names = list(NOMINATIVE = "большой лом", GENITIVE = "большого лома", DATIVE = "большому лому", ACCUSATIVE = "большой лом", INSTRUMENTAL = "большим ломом", PREPOSITIONAL = "большом ломе")
```
Также.

PlayerUnknown14 commented on code/game/objects/items/tools/crowbar.dm:
```diff
@@ -64,17 +66,19 @@
 /obj/item/crowbar/cyborg
 	name = "hydraulic crowbar"
-	desc = "A hydraulic prying tool, compact but powerful. Designed to replace crowbar in construction cyborgs."
+	desc = "Гидравлический лом, компактный, но мощный. Разработан для замены лома в строительных киборгах."
```
`"Разработан для замены лома в строительных киборгах" - "Аналог лома для строительных киборгов."`

PlayerUnknown14 commented on code/game/objects/items/tools/crowbar.dm:
```diff
@@ -64,17 +66,19 @@
 	usesound = 'sound/items/jaws_pry.ogg'
 	force = 10
 	toolspeed = 0.5
+	ru_names = list(NOMINATIVE = "гидравлический лом", GENITIVE = "гидравлического лома", DATIVE = "гидравлическому лому", ACCUSATIVE = "гидравлический лом", INSTRUMENTAL = "гидравлическим ломом", PREPOSITIONAL = "гидравлическом ломе")
```
Также.

PlayerUnknown14 commented on code/game/objects/items/tools/crowbar.dm:
```diff
@@ -64,17 +66,19 @@
 /obj/item/crowbar/power
 	name = "jaws of life"
-	desc = "A set of jaws of life, the magic of science has managed to fit it down into a device small enough to fit in a tool belt. It's fitted with a prying head."
+	desc = "Набор челюстей жизни, магия науки сумела втиснуть его в устройство, достаточно маленькое, чтобы поместиться на поясе для инструментов. Он оснащен выдвигающейся головкой."
```
ё*

PlayerUnknown14 replied:
Переместить сюда `ru_names`.

PlayerUnknown14 commented on code/game/objects/items/tools/crowbar.dm:
```diff
@@ -98,6 +102,6 @@
 	playsound(get_turf(user), 'sound/items/change_jaws.ogg', 50, 1)
 	var/obj/item/wirecutters/power/cutjaws = new /obj/item/wirecutters/power
-	to_chat(user, "<span class='notice'>You attach the cutting jaws to [src].</span>")
+	to_chat(user, "<span class='notice'>Ты присоеденяешь режущую головку к [src.declent_ru(GENITIVE)].</span>")
```
* Обычные информационные сообщения пишем через "вы".
* `"<span class='...'>ТЕКСТ</span>"` заменяем на `span_...("ТЕКСТ")`

`"<span class='notice'>Ты присоеденяешь режущую головку к [src.declent_ru(GENITIVE)].</span>" - span_notice("Вы присоединяете режущую головку к [src.declent_ru(GENITIVE)].")`

Antoonij replied:
```suggestion
	to_chat(user, span_notice("Ты присоединяешь режущую головку к [declent_ru(GENITIVE)]."))
```
src тут не нужен

PlayerUnknown14 commented on code/game/objects/items/tools/holotool.dm:
```diff
@@ -6,14 +6,15 @@ Holotool. All instruments in one object
 /obj/item/holotool
 	name = "experimental holotool"
-	desc = "A highly experimental holographic tool projector."
+	desc = "Высокоэкспериментальный голографический инструмент-проектор."
```
* Слова "Высокоэкспериментальный" в русском языке нет.


`"Высокоэкспериментальный" - "Экспериментальный"`

PlayerUnknown14 commented on code/game/objects/items/tools/holotool.dm:
```diff
@@ -6,14 +6,15 @@ Holotool. All instruments in one object
 	actions_types = list(/datum/action/item_action/change_ht_color)
 	resistance_flags = FIRE_PROOF | ACID_PROOF
 	w_class = WEIGHT_CLASS_SMALL
+	ru_names = list(NOMINATIVE = "эксперементальный голо-тул", GENITIVE = "эксперементального голо-тула", DATIVE = "эксперементальному голо-тулу", ACCUSATIVE = "эксперементальный голо-тул", INSTRUMENTAL = "эксперементальным голо-тулом", PREPOSITIONAL = "эксперементальном голо-туле")
```
Перенести выше, переписать.

Я бы писал `"голо-тул"` слитно, по аналогии со словом `"мультитул"`.

PlayerUnknown14 commented on code/game/objects/items/tools/holotool.dm:
```diff
@@ -24,7 +25,7 @@ Holotool. All instruments in one object
 /obj/item/holotool/examine(mob/user)
 	. = ..()
-	. += span_notice("It is currently set to [current_tool ? current_tool.name : "'off'"] mode.")
+	. += span_notice("На данный момент установлен [current_tool ? current_tool.name : "'off'"] режим.")
```
`"На данный момент установлен off режим."` - Не доделано. Либо переписывать весь код голотула ниже, соотвественно переводя названия инструментов, либо эту строчку откатить и оставить уже другим переводчикам.

PlayerUnknown14 commented on code/game/objects/items/tools/multitool.dm:
```diff
@@ -28,6 +28,7 @@
 	var/shows_wire_information = FALSE // shows what a wire does if set to TRUE
 	var/obj/machinery/buffer // simple machine buffer for device linkage
 	var/datum/multitool_menu_host/menu
+	ru_names = list(NOMINATIVE = "мультитул", GENITIVE = "мультитула", DATIVE = "мультитулу", ACCUSATIVE = "мультитул", INSTRUMENTAL = "мультитулом", PREPOSITIONAL = "мультитуле")
```
Также.

PlayerUnknown14 commented on code/game/objects/items/tools/multitool.dm:
```diff
@@ -43,10 +44,10 @@
 		return
 	buffer = M
-	to_chat(user, "<span class='notice'>You load [M] into [src]'s internal buffer.</span>")
+	to_chat(user, "<span class='notice'>Ты загрузил [M] в внутренний буфер [src.declent_ru(GENITIVE)].</span>")
```
во внутренний*

Переписать на `span_notice`.

PlayerUnknown14 commented on code/game/objects/items/tools/multitool.dm:
```diff
@@ -111,7 +112,7 @@
 
 /obj/item/multitool/ai_detect/admin
-	desc = "Used for pulsing wires to test which to cut. Not recommended by doctors. Has a strange tag that says 'Grief in Safety'" //What else should I say for a meme item?
+	desc = "Используется для пульсации проводов, чтобы проверить, какой из них следует разрезать. Не рекомендуется врачами. Имеет странную бирку с надписью  'Гриф в безопастности'" //What else should I say for a meme item?
```
`'Гриф в безопастности' - 'Гриферить в удовольствие!'`

PlayerUnknown14 commented on code/game/objects/items/tools/multitool.dm:
```diff
@@ -127,22 +128,24 @@
 	toolspeed = 0.1
 	origin_tech = "magnets=5;engineering=5;abductor=3"
 	shows_wire_information = TRUE
+	ru_names = list(NOMINATIVE = "инопланетный мультитул", GENITIVE = "инопланетного мультитула", DATIVE = "инопланетному мультитулу", ACCUSATIVE = "инопланетный мультитул", INSTRUMENTAL = "инопланетным мультитулом", PREPOSITIONAL = "инопланетном мультитуле")
```
Также.

PlayerUnknown14 commented on code/game/objects/items/tools/multitool.dm:
```diff
@@ -127,22 +128,24 @@
 	icon_state = "multitool_brass"
 	toolspeed = 0.5
 	resistance_flags = FIRE_PROOF | ACID_PROOF
+	ru_names = list(NOMINATIVE = "латунный мультитул", GENITIVE = "латунного мультитула", DATIVE = "латунному мультитулу", ACCUSATIVE = "латунный мультитул", INSTRUMENTAL = "латунным мультитулом", PREPOSITIONAL = "латунном мультитуле")
```
Также.

PlayerUnknown14 commented on code/game/objects/items/tools/screwdriver.dm:
```diff
@@ -1,7 +1,7 @@
 /obj/item/screwdriver
 	name = "screwdriver"
-	desc = "You can be totally screwy with this."
+	desc = "С этим можно быть совершенно чокнутым."
```
* Непереводимая игра слов, пытаться перевести прямо - не вариант.


`"С этим можно быть совершенно чокнутым." - "Инструмент для откручивания и прикручивания различных предметов. Плохо сочетается с глазными яблоками."`

PlayerUnknown14 commented on code/game/objects/items/tools/screwdriver.dm:
```diff
@@ -22,20 +22,21 @@
 	armor = list("melee" = 0, "bullet" = 0, "laser" = 0, "energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 50, "acid" = 30)
 	tool_behaviour = TOOL_SCREWDRIVER
 	var/random_color = TRUE //if the screwdriver uses random coloring
+	ru_names = list(NOMINATIVE = "отвертка", GENITIVE = "отвертки", DATIVE = "отвертке", ACCUSATIVE = "отвертку", INSTRUMENTAL = "отверткой", PREPOSITIONAL = "отвертке")
```
ё*

Также.

PlayerUnknown14 commented on code/game/objects/items/tools/screwdriver.dm:
```diff
@@ -22,20 +22,21 @@
 /obj/item/screwdriver/nuke
 	name = "screwdriver"
-	desc = "A screwdriver with an ultra thin tip."
+	desc = "Отвертка с ультратонким наконечником."
```
ё*

Прописать `ru_names`.

Dimempers replied:
Тут такое же самое название как и у оригинальной, можно просто скопировать ру нейм с оригинальной?

PlayerUnknown14 commented on code/game/objects/items/tools/screwdriver.dm:
```diff
@@ -22,20 +22,21 @@
 
 /obj/item/screwdriver/suicide_act(mob/user)
-	user.visible_message("<span class='suicide'>[user] is stabbing [src] into [user.p_their()] [pick("temple", "heart")]! It looks like [user.p_theyre()] trying to commit suicide!</span>")
+	user.visible_message("<span class='suicide'>[user] кол[pluralize_ru(user.gender,"ет","ют")] [src.declent_ru(INSTRUMENTAL)] в [user.p_their()] [pick("temple", "heart")]! Похоже на то, что [user.p_theyre()] пыта[pluralize_ru(user.gender,"ется","ются")] совершить самоубийство!</span>")
```
`span_suicide`

`"[user] кол[pluralize_ru(user.gender,"ет","ют")] [src.declent_ru(INSTRUMENTAL)] в [user.p_their()] [pick("temple", "heart")]! Похоже на то, что [user.p_theyre()] пыта[pluralize_ru(user.gender,"ется","ются")] совершить самоубийство!" - "[user] кол[pluralize_ru(user.gender,"ет","ют")] себя [src.declent_ru(INSTRUMENTAL)] в [pick("висок", "сердце")]! Похоже на то, что [genderize_ru(user.gender, "он", "она", "оно", "они")] пыта[pluralize_ru(user.gender,"ет","ют")]ся совершить самоубийство!"`

PlayerUnknown14 commented on code/game/objects/items/tools/screwdriver.dm:
```diff
@@ -65,15 +66,15 @@
 	name = "brass screwdriver"
-	desc = "A screwdriver made of brass. The handle feels freezing cold."
+	desc = "Отвертка из латуни. Ручка кажется ледяной."
 	icon_state = "screwdriver_brass"
```
`ru_names`

ё*

PlayerUnknown14 commented on code/game/objects/items/tools/screwdriver.dm:
```diff
@@ -65,15 +66,15 @@
 /obj/item/screwdriver/abductor
 	name = "alien screwdriver"
-	desc = "An ultrasonic screwdriver."
+	desc = "Ультразвуковая отвертка."
```
Также.

PlayerUnknown14 commented on code/game/objects/items/tools/screwdriver.dm:
```diff
@@ -98,24 +99,25 @@
 
 /obj/item/screwdriver/power/suicide_act(mob/user)
-	user.visible_message("<span class='suicide'>[user] is putting [src] to [user.p_their()] temple. It looks like [user.p_theyre()] trying to commit suicide!</span>")
+	user.visible_message("<span class='suicide'>[user] ставит [src.declent_ru(ACCUSATIVE)] к [genderize_ru(user.gender,"его","её","его","их")] виску. Похоже [genderize_ru(user.gender,"он","она","оно","они")] пыта[pluralize_ru(user.gender,"ется","ются")] совершить самоубийство!</span>")
```
Переписать аналогично этому сообщению выше.

Antoonij replied:
```suggestion
	user.visible_message(span_suicide("[user] ставит [declent_ru(ACCUSATIVE)] к [genderize_ru(user.gender, "его", "её", "его", "их")] виску. Похоже [genderize_ru(user.gender, "он", "она", "оно", "они")] пыта[pluralize_ru(user.gender, "ется", "ются")] совершить самоубийство!"))
```
Пробелы после запятых, src тут не нужен

PlayerUnknown14 commented on code/game/objects/items/tools/screwdriver.dm:
```diff
@@ -98,24 +99,25 @@
 	playsound(get_turf(user), 'sound/items/change_drill.ogg', 50, 1)
 	var/obj/item/wrench/power/b_drill = new /obj/item/wrench/power
-	to_chat(user, "<span class='notice'>You attach the bolt driver bit to [src].</span>")
+	to_chat(user, "<span class='notice'>Ты присоеденяешь головку болтового сверла к [src.declent_ru(GENITIVE)].</span>")
```
Вы*

PlayerUnknown14 commented on code/game/objects/items/tools/screwdriver.dm:
```diff
@@ -98,24 +99,25 @@
 /obj/item/screwdriver/cyborg
 	name = "powered screwdriver"
-	desc = "An electrical screwdriver, designed to be both precise and quick."
+	desc = "Электрическая отвертка, разработанная для точного и быстрого использования."
```
Также.

PlayerUnknown14 commented on code/game/objects/items/tools/welder.dm:
```diff
@@ -39,6 +39,7 @@
 	light_power = 0.75
 	light_color = LIGHT_COLOR_FIRE
 	light_on = FALSE
+	ru_names = list(NOMINATIVE = "сварочный аппарат", GENITIVE = "сварочного аппарата", DATIVE = "сварочному аппарату", ACCUSATIVE = "сварочный аппарат", INSTRUMENTAL = "сварочным аппаратом", PREPOSITIONAL = "сварочном аппарате")
```
Также.

PlayerUnknown14 commented on code/game/objects/items/tools/welder.dm:
```diff
@@ -54,15 +55,15 @@
 	. = ..()
 	if(get_dist(user, src) <= 0)
-		. += "<span class='notice'>It contains [GET_FUEL] unit\s of fuel out of [maximum_fuel].</span>"
+		. += "<span class='notice'>Он содержит [GET_FUEL] юнит/ов топлива из [maximum_fuel].</span>"
```
`"юнит/ов" - "u"`

`span_notice`

PlayerUnknown14 commented on code/game/objects/items/tools/welder.dm:
```diff
@@ -54,15 +55,15 @@
 
 /obj/item/weldingtool/suicide_act(mob/user)
-	user.visible_message("<span class='suicide'>[user] welds [user.p_their()] every orifice closed! It looks like [user.p_theyre()] trying to commit suicide!</span>")
+	user.visible_message("<span class='suicide'>[user] заварива[pluralize_ru(user.gender,"ет","ют")] себе все отверстия! Похоже [genderize_ru(user.gender,"он","она","оно","они")] пыта[pluralize_ru(user.gender,"ется","ются")] совершить самоубийство!</span>")
```
Аналогично выше.

PlayerUnknown14 commented on code/game/objects/items/tools/welder.dm:
```diff
@@ -54,15 +55,15 @@
 /obj/item/weldingtool/can_enter_storage(obj/item/storage/S, mob/user)
 	if(tool_enabled)
-		to_chat(user, "<span class='warning'>[S] can't hold [src] while it's lit!</span>")
+		to_chat(user, "<span class='warning'>[S] не могу положить [src.declent_ru(NOMINATIVE)] пока он горит!</span>")
```
`"Контейнер не могу положить сварочный аппарат пока он горит!"`

Переписать + `span_warning`.

Antoonij replied:
```suggestion
		to_chat(user, span_warning("[S] не могу положить [declent_ru(NOMINATIVE)] пока он горит!"))
```
src тут не нужен

PlayerUnknown14 commented on code/game/objects/items/tools/welder.dm:
```diff
@@ -88,14 +89,14 @@
 /obj/item/weldingtool/attack_self(mob/user)
 	if(tool_enabled) //Turn off the welder if it's on
-		to_chat(user, "<span class='notice'>You switch off [src].</span>")
+		to_chat(user, "<span class='notice'>Ты выключил [src.declent_ru(NOMINATIVE)].</span>")
```
Вы* здесь и далее.

PlayerUnknown14 commented on code/game/objects/items/tools/welder.dm:
```diff
@@ -124,13 +125,13 @@
 	if(!tool_enabled)
 		if(!silent)
-			to_chat(user, "<span class='notice'>[src] has to be on to complete this task!</span>")
+			to_chat(user, "<span class='notice'>[src.declent_ru(NOMINATIVE)] должен быть включен для выполнения этой задачи!</span>")
```
ё*

PlayerUnknown14 commented on code/game/objects/items/tools/welder.dm:
```diff
@@ -171,16 +172,16 @@
 		return
 	if(GET_FUEL >= maximum_fuel)
-		to_chat(user, "<span class='notice'>[src] is already full!</span>")
+		to_chat(user, "<span class='notice'>[src.declent_ru(NOMINATIVE)] уже полон!</span>")
```
Если declent_ru используется в начале строки, то нам нужна заглавная буква.

`[src.declent_ru(NOMINATIVE)] - [capitalize(src.declent_ru(NOMINATIVE))]`

PlayerUnknown14 commented on code/game/objects/items/tools/welder.dm:
```diff
@@ -171,16 +172,16 @@
 	var/amount_transferred = A.reagents.trans_id_to(src, "fuel", amount)
 	if(amount_transferred)
-		to_chat(user, "<span class='notice'>You refuel [src] by [amount_transferred] unit\s.</span>")
+		to_chat(user, "<span class='notice'>Ты заполнил [src.declent_ru(NOMINATIVE)] [amount_transferred] юнитами.</span>")
```
Аналогично выше.

PlayerUnknown14 commented on code/game/objects/items/tools/welder.dm:
```diff
@@ -171,16 +172,16 @@
 		return amount_transferred
 	else
-		to_chat(user, "<span class='warning'>There's not enough fuel in [A] to refuel [src]!</span>")
+		to_chat(user, "<span class='warning'>В [A] недостатачно топлива для заполнения [src.declent_ru(GENITIVE)]!</span>")
```
Лучше перепиши на балун.

PlayerUnknown14 commented on code/game/objects/items/tools/welder.dm:
```diff
@@ -205,7 +206,7 @@
 /obj/item/weldingtool/largetank
 	name = "industrial welding tool"
-	desc = "A slightly larger welder with a larger tank."
+	desc = "Немного более крупный сварочный аппарат с большим баком."
```
`"Немного более крупный сварочный аппарат с большим баком." - "Сварочный аппарат, но с баком побольше."`

PlayerUnknown14 replied:
`ru_names`

PlayerUnknown14 commented on code/game/objects/items/tools/welder.dm:
```diff
@@ -214,12 +215,12 @@
 /obj/item/weldingtool/largetank/cyborg
 	name = "integrated welding tool"
-	desc = "An advanced welder designed to be used in robotic systems."
+	desc = "Усовершенствованный сварочный аппарат, предназначенный для использования в роботизированных системах."
```
Также.

PlayerUnknown14 commented on code/game/objects/items/tools/welder.dm:
```diff
@@ -214,12 +215,12 @@
 /obj/item/weldingtool/mini
 	name = "emergency welding tool"
-	desc = "A miniature welder used during emergencies."
+	desc = "Миниатюрный сварочный аппарат, используемый в экстренных случаях."
```
Также.

PlayerUnknown14 commented on code/game/objects/items/tools/welder.dm:
```diff
@@ -228,7 +229,7 @@
 /obj/item/weldingtool/abductor
 	name = "alien welding tool"
-	desc = "An alien welding tool. Whatever fuel it uses, it never runs out."
+	desc = "Инопланетный сварочный инструмент. Какое бы топливо он ни использовал, оно никогда не кончается."
```
Также.

PlayerUnknown14 commented on code/game/objects/items/tools/welder.dm:
```diff
@@ -242,7 +243,7 @@
 /obj/item/weldingtool/hugetank
 	name = "upgraded welding tool"
-	desc = "An upgraded welder based off the industrial welder."
+	desc = "Усовершенствованный сварочный аппарат на базе промышленного сварочного аппарата."
```
Также.

PlayerUnknown14 commented on code/game/objects/items/tools/welder.dm:
```diff
@@ -252,7 +253,7 @@
 /obj/item/weldingtool/experimental
 	name = "experimental welding tool"
-	desc = "An experimental welder capable of self-fuel generation and less harmful to the eyes."
+	desc = "Экспериментальный сварочный аппарат, способный самостоятельно вырабатывать топливо и менее вредный для глаз."
```
Также.

PlayerUnknown14 commented on code/game/objects/items/tools/welder.dm:
```diff
@@ -266,13 +267,13 @@
 /obj/item/weldingtool/experimental/mecha
 	name = "integrated welding tool"
-	desc = "An advanced welder designed to be used in robotic systems."
+	desc = "Усовершенствованный сварочный аппарат, предназначенный для использования в роботизированных системах."
```
Также.

PlayerUnknown14 commented on code/game/objects/items/tools/welder.dm:
```diff
@@ -266,13 +267,13 @@
 /obj/item/weldingtool/experimental/brass
 	name = "brass welding tool"
-	desc = "A brass welder that seems to constantly refuel itself. It is faintly warm to the touch."
+	desc = "Латунный сварочный аппарат, который, кажется, постоянно заправляется. Он слегка теплый на ощупь."
```
Также.

PlayerUnknown14 commented on code/game/objects/items/tools/wirecutters.dm:
```diff
@@ -1,6 +1,6 @@
 /obj/item/wirecutters
 	name = "wirecutters"
-	desc = "This cuts wires."
+	desc = "Это перерезает провода."
```
`"Инструмент для перерезания различных материалов."`

PlayerUnknown14 commented on code/game/objects/items/tools/wirecutters.dm:
```diff
@@ -24,6 +24,7 @@
 	armor = list("melee" = 0, "bullet" = 0, "laser" = 0, "energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 50, "acid" = 30)
 	tool_behaviour = TOOL_WIRECUTTER
 	var/random_color = TRUE
+	ru_names = list(NOMINATIVE = "кусачки", GENITIVE = "кусачек", DATIVE = "кусачкам", ACCUSATIVE = "кусачки", INSTRUMENTAL = "кусачками", PREPOSITIONAL = "кусачках")
```
Также.

PlayerUnknown14 commented on code/game/objects/items/tools/wirecutters.dm:
```diff
@@ -41,8 +42,8 @@
-			span_notice("[user] cuts [target]'s restraints with [src]!"),
-			span_notice("You have cut [target]'s restraints with [src]!"),
+			span_notice("[user] режет [target]'s стяжки [src.declent_ru(INSTRUMENTAL)]!"),
+			span_notice("Ты перерезал [target]'s стяжки [src.declent_ru(INSTRUMENTAL)]!"),
```
Переписать.

PlayerUnknown14 commented on code/game/objects/items/tools/wirecutters.dm:
```diff
@@ -52,21 +53,21 @@
 
 /obj/item/wirecutters/suicide_act(mob/user)
-	user.visible_message("<span class='suicide'>[user] is cutting at [user.p_their()] arteries with [src]! It looks like [user.p_theyre()] trying to commit suicide!</span>")
+	user.visible_message("<span class='suicide'>[user] перереза[pluralize_ru(user.gender,"ет","ют")] [genderize_ru(user.gender,"его","её","его","их")] артерии с помощью [src.declent_ru(GENITIVE)]! Кажется, [genderize_ru(user.gender,"он","она","оно","они")] пыта[pluralize_ru(user.gender,"ется","ются")] совершить самоубийство!</span>")
```
Также.

PlayerUnknown14 commented on code/game/objects/items/tools/wirecutters.dm:
```diff
@@ -52,21 +53,21 @@
 /obj/item/wirecutters/brass
 	name = "brass wirecutters"
-	desc = "A pair of wirecutters made of brass. The handle feels freezing cold to the touch."
+	desc = "Пара кусачек из латуни. Ручка на ощупь ледяная."
```
Также.

PlayerUnknown14 commented on code/game/objects/items/tools/wirecutters.dm:
```diff
@@ -52,21 +53,21 @@
 /obj/item/wirecutters/abductor
 	name = "alien wirecutters"
-	desc = "Extremely sharp wirecutters, made out of a silvery-green metal."
+	desc = "Очень острые кусачки, изготовленные из серебристо-зеленого металла."
```
Также.

PlayerUnknown14 commented on code/game/objects/items/tools/wirecutters.dm:
```diff
@@ -77,12 +78,12 @@
 /obj/item/wirecutters/cyborg
 	name = "wirecutters"
-	desc = "This cuts wires."
+	desc = "Это перерезает провода."
```
Также.

PlayerUnknown14 commented on code/game/objects/items/tools/wirecutters.dm:
```diff
@@ -91,9 +92,11 @@
 	usesound = 'sound/items/jaws_cut.ogg'
 	toolspeed = 0.25
 	random_color = FALSE
+	ru_names = list(NOMINATIVE = "челюсти жизни", GENITIVE = "челюстей жизни", DATIVE = "челюстям жизни", ACCUSATIVE = "челюсти жизни", INSTRUMENTAL = "челюстями жизни", PREPOSITIONAL = "челюстях жизни")
```
Также.

PlayerUnknown14 commented on code/game/objects/items/tools/wirecutters.dm:
```diff
@@ -91,9 +92,11 @@
 
 /obj/item/wirecutters/power/suicide_act(mob/user)
-	user.visible_message("<span class='suicide'>[user] is wrapping \the [src] around [user.p_their()] neck. It looks like [user.p_theyre()] trying to rip [user.p_their()] head off!</span>")
+	user.visible_message("<span class='suicide'>[user] обертывает [src.declent_ru(NOMINATIVE)] вокруг [genderize_ru(user.gender,"его","её","его","их")] шеи. Кажется, [genderize_ru(user.gender,"он","она","оно","они")] пыта[pluralize_ru(user.gender,"ется","ются")] оторвать [genderize_ru(user.gender,"его","её","его","их")] голову!</span>")
```
Также.

PlayerUnknown14 commented on code/game/objects/items/tools/wirecutters.dm:
```diff
@@ -106,6 +109,6 @@
 	playsound(get_turf(user), 'sound/items/change_jaws.ogg', 50, 1)
 	var/obj/item/crowbar/power/pryjaws = new /obj/item/crowbar/power
-	to_chat(user, "<span class='notice'>You attach the pry jaws to [src].</span>")
+	to_chat(user, "<span class='notice'Ты присоеденяешь поддевающую головку к [src.declent_ru(DATIVE)].</span>")
```
Также.

PlayerUnknown14 commented on code/game/objects/items/tools/wrench.dm:
```diff
@@ -19,31 +19,32 @@
 
 /obj/item/wrench/suicide_act(mob/user)
-	user.visible_message("<span class='suicide'>[user] is beating [user.p_them()]self to death with [src]! It looks like [user.p_theyre()] trying to commit suicide!</span>")
+	user.visible_message("<span class='suicide'>[user] избивает себя до смерти [src.declent_ru(INSTRUMENTAL)]! Похоже, [genderize_ru(user.gender,"он","она","оно","они")] пыта[pluralize_ru(user.gender,"ется","ются")] совершить самоубийсто!</span>")
```
Также.

PlayerUnknown14 commented on code/game/objects/items/tools/wrench.dm:
```diff
@@ -19,31 +19,32 @@
 /obj/item/wrench/cyborg
 	name = "automatic wrench"
-	desc = "An advanced robotic wrench. Can be found in construction cyborgs."
+	desc = "Продвинутый роботизированный гаечный ключ. Можно найти у строительных киборгов."
```
Также.

PlayerUnknown14 commented on code/game/objects/items/tools/wrench.dm:
```diff
@@ -19,31 +19,32 @@
 /obj/item/wrench/brass
 	name = "brass wrench"
-	desc = "A brass wrench. It's faintly warm to the touch."
+	desc = "Латунный ключ. Он слегка теплый на ощупь."
```
Также.

PlayerUnknown14 commented on code/game/objects/items/tools/wrench.dm:
```diff
@@ -19,31 +19,32 @@
 /obj/item/wrench/abductor
 	name = "alien wrench"
-	desc = "A polarized wrench. It causes anything placed between the jaws to turn."
+	desc = "Поляризованный ключ. Он заставляет все, что находится между губками, поворачиваться."
```
Также.

PlayerUnknown14 commented on code/game/objects/items/tools/wrench.dm:
```diff
@@ -54,7 +55,7 @@
 /obj/item/wrench/power
 	name = "hand drill"
-	desc = "A simple powered drill with a bolt bit."
+	desc = "Простая электрическая дрель с болтовым сверлом."
```
Также.

PlayerUnknown14 commented on code/game/objects/items/tools/wrench.dm:
```diff
@@ -65,29 +66,31 @@
 	throwforce = 8
 	attack_verb = list("drilled", "screwed", "jabbed")
 	toolspeed = 0.25
+	ru_names = list(NOMINATIVE = "ручная дрель", GENITIVE = "ручной дрели", DATIVE = "ручную дрель", ACCUSATIVE = "ручную дрель", INSTRUMENTAL = "ручной дрелью", PREPOSITIONAL = "ручной дрели")
```
Также.

PlayerUnknown14 commented on code/game/objects/items/tools/wrench.dm:
```diff
@@ -65,29 +66,31 @@
 	playsound(get_turf(user),'sound/items/change_drill.ogg', 50, 1)
 	var/obj/item/wirecutters/power/s_drill = new /obj/item/screwdriver/power
-	to_chat(user, "<span class='notice'>You attach the screwdriver bit to [src].</span>")
+	to_chat(user, "<span class='notice'>Ты присоеденяешь головку отвертки к [src.declent_ru(GENITIVE)].</span>")
```
Также.

PlayerUnknown14 commented on code/game/objects/items/tools/wrench.dm:
```diff
@@ -65,29 +66,31 @@
 
 /obj/item/wrench/power/suicide_act(mob/user)
-	user.visible_message("<span class='suicide'>[user] is pressing [src] against [user.p_their()] head! It looks like [user.p_theyre()] trying to commit suicide!")
+	user.visible_message("<span class='suicide'>[user] нажимает [src.declent_ru(INSTRUMENTAL)] против [genderize_ru(user.gender,"его","её","его","их")] головы! Похоже, [genderize_ru(user.gender,"он","она","оно","они")] пыта[pluralize_ru(user.gender,"ется","ются")] совершить самоубийство!")
```
Также.

PlayerUnknown14 commented on code/game/objects/items/tools/wrench.dm:
```diff
@@ -65,29 +66,31 @@
 /obj/item/wrench/medical
 	name = "medical wrench"
-	desc = "A medical wrench with common (medical?) uses. Can be found in your hand."
+	desc = "Медицинский ключ с общим (медицинским?) Использование. Можно найти в руке."
```
Переписать описание.

PlayerUnknown14 commented on code/game/objects/items/tools/wrench.dm:
```diff
@@ -65,29 +66,31 @@
 	throwforce = 4
 	origin_tech = "materials=1;engineering=1;biotech=3"
 	attack_verb = list("wrenched", "medicaled", "tapped", "jabbed", "whacked")
+	ru_names = list(NOMINATIVE = "медецинский ключ", GENITIVE = "медецинского ключа", DATIVE = "медецинскому ключу", ACCUSATIVE = "медецинской ключ", INSTRUMENTAL = "медецинским ключем", PREPOSITIONAL = "медецинском ключе")
```
Также.

PlayerUnknown14 commented on code/game/objects/items/tools/wrench.dm:
```diff
@@ -65,29 +66,31 @@
 
 /obj/item/wrench/medical/suicide_act(mob/living/user)
-	user.visible_message("<span class='suicide'>[user] is praying to the medical wrench to take [user.p_their()] soul. It looks like [user.p_theyre()] trying to commit suicide!</span>")
+	user.visible_message("<span class='suicide'>[user] молится медецинскому ключу чтобы он взял [genderize_ru(user.gender,"его","её","его","их")] душу. Похоже, [genderize_ru(user.gender,"он","она","оно","они")] пыта[pluralize_ru(user.gender,"ется","ются")] совершить самоубийство!</span>")
```
Также.

PlayerUnknown14 reviewed (commented):
Дописать.

Antoonij commented on code/game/objects/items/tools/screwdriver.dm:
```diff
@@ -98,24 +99,25 @@
 	playsound(get_turf(user), 'sound/items/change_drill.ogg', 50, 1)
 	var/obj/item/wrench/power/b_drill = new /obj/item/wrench/power
-	to_chat(user, "<span class='notice'>You attach the bolt driver bit to [src].</span>")
+	to_chat(user, "<span class='notice'>Ты присоеденяешь головку болтового сверла к [src.declent_ru(GENITIVE)].</span>")
```
```suggestion
	to_chat(user, span_notice("Вы присоединяете головку болтового сверла к [declent_ru(GENITIVE)]."))
```
src тут не нужен

Antoonij replied:
и далее посмотришь сам, где у тебя src.declent_ru, src. оттуда нужно убрать

Dimempers committed:
Merge branch 'Translation-of-engineering-tools' of https://github.com/Dimempers/Paradise into Translation-of-engineering-tools

Dimempers committed:
Correction of errors and shortcomings

Dimempers committed:
fixes errors and warnings

Dimempers committed:
again fix error

Забыл прописать list в ru_names, поправил

Dimempers committed:
Поправка падежа в джавсах

Случайно поставил не тот падеж в переключении насадки, исправил

Dimempers committed:
Merge branch 'master220' into Translation-of-engineering-tools

PlayerUnknown14 commented on code/game/objects/items/tools/crowbar.dm:
```diff
@@ -1,6 +1,14 @@
 /obj/item/crowbar
 	name = "pocket crowbar"
-	desc = "Небольшой лом. Этот удобный инструмент полезен для многих вещей, таких как поддевание напольной плитки или открытие дверей без электропитания."
+	desc = "Небольшой лом. Этот удобный инструмент полезен для многих вещей, таких как поддевание напольной плитки или открытие обесточеных шлюзов."
```
обесточеННых*

PlayerUnknown14 commented on code/game/objects/items/tools/crowbar.dm:
```diff
@@ -87,21 +130,20 @@
 
 /obj/item/crowbar/power/suicide_act(mob/user)
-	user.visible_message("<span class='suicide'>[user] помеща[pluralize_ru(user.gender,"ет","ют")] свою голову между лезвиями [src.declent_ru(GENITIVE)]. Похоже, [genderize_ru(user.gender,"он","она","оно","они")] пыта[pluralize_ru(user.gender,"ется","ются")] использовать [src.declent_ru(ACCUSATIVE)] для самоубийства!</span>")
+	user.visible_message(span_suicide("[user] помеща[pluralize_ru(user.gender, "ет", "ют")] свою голову между лезвиями [declent_ru(GENITIVE)]. Похоже, [genderize_ru(user.gender, "он", "она", "оно", "они")] пыта[pluralize_ru(user.gender, "ет", "ют")]ся использовать [declent_ru(ACCUSATIVE)] для самоубийства!"))
```
"Похоже, что"

PlayerUnknown14 commented on code/game/objects/items/tools/holotool.dm:
```diff
@@ -6,16 +6,14 @@ Holotool. All instruments in one object
 /obj/item/holotool
 	name = "experimental holotool"
-	desc = "Высокоэкспериментальный голографический инструмент-проектор."
+	desc = "A highly experimental holographic tool projector."
```
Ты немного не в ту сторону переводишь.

PlayerUnknown14 commented on code/game/objects/items/tools/holotool.dm:
```diff
@@ -6,16 +6,14 @@ Holotool. All instruments in one object
 	actions_types = list(/datum/action/item_action/change_ht_color)
 	resistance_flags = FIRE_PROOF | ACID_PROOF
 	w_class = WEIGHT_CLASS_SMALL
-	ru_names = list(NOMINATIVE = "эксперементальный голо-тул", GENITIVE = "эксперементального голо-тула", DATIVE = "эксперементальному голо-тулу", ACCUSATIVE = "эксперементальный голо-тул", INSTRUMENTAL = "эксперементальным голо-тулом", PREPOSITIONAL = "эксперементальном голо-туле")
```
А тут удалил строчку вообще.

PlayerUnknown14 commented on code/game/objects/items/tools/holotool.dm:
```diff
@@ -25,7 +23,7 @@ Holotool. All instruments in one object
 /obj/item/holotool/examine(mob/user)
 	. = ..()
-	. += span_notice("На данный момент установлен [current_tool ? current_tool.name : "'off'"] режим.")
+	. += (span_notice("It is currently set to [current_tool ? current_tool.name : "'off'"] mode."))
```
И тут.

Dimempers replied:
Оставил другим

PlayerUnknown14 commented on code/game/objects/items/tools/multitool.dm:
```diff
@@ -47,7 +54,7 @@
 		return
 	buffer = M
-	to_chat(user, "<span class='notice'>Ты загрузил [M] в внутренний буфер [src.declent_ru(GENITIVE)].</span>")
+	to_chat(user, span_notice("Ты загрузил [M] во внутренний буфер [declent_ru(GENITIVE)]."))
```
Во всех подобных сообщениях должно быть "вы".

PlayerUnknown14 commented on code/game/objects/items/tools/screwdriver.dm:
```diff
@@ -99,25 +130,24 @@
 
 /obj/item/screwdriver/power/suicide_act(mob/user)
-	user.visible_message("<span class='suicide'>[user] ставит [src.declent_ru(ACCUSATIVE)] к [genderize_ru(user.gender,"его","её","его","их")] виску. Похоже [genderize_ru(user.gender,"он","она","оно","они")] пыта[pluralize_ru(user.gender,"ется","ются")] совершить самоубийство!</span>")
+	user.visible_message(span_suicide("[user] став[pluralize_ru(user.gender, "ит", "ят")] [declent_ru(ACCUSATIVE)] к своему виску. Похоже [genderize_ru(user.gender, "он", "она", "оно", "они")] пыта[pluralize_ru(user.gender, "ет", "ют")]ся совершить самоубийство!"))
```
`"ставит" - "приставляет"`

`"Похоже, что"`

PlayerUnknown14 commented on code/game/objects/items/tools/screwdriver.dm:
```diff
@@ -99,25 +130,24 @@
 	playsound(get_turf(user), 'sound/items/change_drill.ogg', 50, 1)
 	var/obj/item/wrench/power/b_drill = new /obj/item/wrench/power
-	to_chat(user, "<span class='notice'>Ты присоеденяешь головку болтового сверла к [src.declent_ru(GENITIVE)].</span>")
+	to_chat(user, span_notice("Вы присоеденяете головку болтового сверла к [declent_ru(GENITIVE)]."))
```
Падеж здесь - дательный.

PlayerUnknown14 commented on code/game/objects/items/tools/screwdriver.dm:
```diff
@@ -99,25 +130,24 @@
 /obj/item/screwdriver/cyborg
 	name = "powered screwdriver"
-	desc = "Электрическая отвертка, разработанная для точного и быстрого использования."
+	desc = "Электрическая отвёртка, разработанная для точного и быстрого использования."
```
"Электрическая отвёртка" - "Отвёртка с электроприводом."

PlayerUnknown14 commented on code/game/objects/items/tools/welder.dm:
```diff
@@ -55,15 +62,15 @@
 /obj/item/weldingtool/can_enter_storage(obj/item/storage/S, mob/user)
 	if(tool_enabled)
-		to_chat(user, "<span class='warning'>[S] не могу положить [src.declent_ru(NOMINATIVE)] пока он горит!</span>")
+		to_chat(user, span_warning("[capitalize(declent_ru(NOMINATIVE))] невозможно положить пока он включен!"))
```
включЁн*

положить, пока

Падеж - винительный.

PlayerUnknown14 commented on code/game/objects/items/tools/welder.dm:
```diff
@@ -89,14 +96,14 @@
 /obj/item/weldingtool/attack_self(mob/user)
 	if(tool_enabled) //Turn off the welder if it's on
-		to_chat(user, "<span class='notice'>Ты выключил [src.declent_ru(NOMINATIVE)].</span>")
+		to_chat(user, span_notice("Вы выключили [declent_ru(NOMINATIVE)]."))
```
Падеж здесь и ниже - родительный.

А вообще, перенеси в балуны эти сообщения.

Dimempers replied:
С родительным это будет "Вы выключили сварочного аппарата", переписал на винительный.

PlayerUnknown14 commented on code/game/objects/items/tools/welder.dm:
```diff
@@ -125,13 +132,13 @@
 	if(!tool_enabled)
 		if(!silent)
-			to_chat(user, "<span class='notice'>[src.declent_ru(NOMINATIVE)] должен быть включен для выполнения этой задачи!</span>")
+			to_chat(user, span_notice("[capitalize(declent_ru(NOMINATIVE))] должен быть включён для выполнения этой задачи!"))
```
В балун.

PlayerUnknown14 commented on code/game/objects/items/tools/welder.dm:
```diff
@@ -125,13 +132,13 @@
 	else
 		if(!silent)
-			to_chat(user, "<span class='warning'>Тебе нужно больше сварочного топлива в [src.declent_ru(PREPOSITIONAL)] для выполнения этой задачи!</span>")
+			to_chat(user, span_warning("Тебе нужно больше сварочного топлива в [declent_ru(PREPOSITIONAL)] для выполнения этой задачи!"))
```
В балун.

PlayerUnknown14 commented on code/game/objects/items/tools/welder.dm:
```diff
@@ -172,16 +179,16 @@
 		return
 	if(GET_FUEL >= maximum_fuel)
-		to_chat(user, "<span class='notice'>[src.declent_ru(NOMINATIVE)] уже полон!</span>")
+		to_chat(user, span_notice("[capitalize(declent_ru(NOMINATIVE))] уже полон!"))
```
В балун.

PlayerUnknown14 commented on code/game/objects/items/tools/welder.dm:
```diff
@@ -172,16 +179,16 @@
 	var/amount_transferred = A.reagents.trans_id_to(src, "fuel", amount)
 	if(amount_transferred)
-		to_chat(user, "<span class='notice'>Ты заполнил [src.declent_ru(NOMINATIVE)] [amount_transferred] юнитами.</span>")
+		to_chat(user, span_notice("Ты заполнил [declent_ru(NOMINATIVE)] [amount_transferred] u.</span>"))
```
.

PlayerUnknown14 commented on code/game/objects/items/tools/welder.dm:
```diff
@@ -172,16 +179,16 @@
 		return amount_transferred
 	else
-		to_chat(user, "<span class='warning'>В [A] недостатачно топлива для заполнения [src.declent_ru(GENITIVE)]!</span>")
+		balloon_alert(user, span_warning("В [A] недостатачно топлива!</span>"))
```
.

PlayerUnknown14 commented on code/game/objects/items/tools/welder.dm:
```diff
@@ -216,11 +231,27 @@
+			DATIVE = "встроенному сварочному аппарату",
+			ACCUSATIVE = "встроенный сварочный аппарат",
+			INSTRUMENTAL = "встроенным сварочным аппаратом",
+			PREPOSITIONAL = "встроенном сварачном аппарате"
```
сварАчном)))

PlayerUnknown14 commented on code/game/objects/items/tools/welder.dm:
```diff
@@ -244,6 +283,14 @@
 /obj/item/weldingtool/hugetank
 	name = "upgraded welding tool"
 	desc = "Усовершенствованный сварочный аппарат на базе промышленного сварочного аппарата."
```
`"Усовершенствованный сварочный аппарат на базе промышленного сварочного аппарата." - "Усовершенствованная версия промышленного сварочного аппарата. С баком побольше!"`

PlayerUnknown14 commented on code/game/objects/items/tools/welder.dm:
```diff
@@ -268,12 +323,28 @@
+			DATIVE = "встроенному сварочному аппарату",
+			ACCUSATIVE = "встроенный сварочный аппарат",
+			INSTRUMENTAL = "встроенным сварочным аппаратом",
+			PREPOSITIONAL = "встроенном сварачном аппарате"
```
сварАчном)))

PlayerUnknown14 commented on code/game/objects/items/tools/wirecutters.dm:
```diff
@@ -42,8 +49,8 @@
-			span_notice("Ты перерезал [target]'s стяжки [src.declent_ru(INSTRUMENTAL)]!"),
+			span_notice("[user] перерезает стяжки [target]'s с помощью [declent_ru(GENITIVE)]!"),
+			span_notice("Ты перерезал стяжки [target]'s с помощью [declent_ru(GENITIVE)]!"),
 		)
```
```
visible_message(span_notice("[user] перереза[pluralize_ru(user.gender, "ет", "ют")] стяжки [target] с помощью [declent_ru(GENITIVE)]!"))

balloon_alert(user, "стяжки перерезаны")
```

PlayerUnknown14 commented on code/game/objects/items/tools/wirecutters.dm:
```diff
@@ -53,13 +60,21 @@
 
 /obj/item/wirecutters/suicide_act(mob/user)
-	user.visible_message("<span class='suicide'>[user] перереза[pluralize_ru(user.gender,"ет","ют")] [genderize_ru(user.gender,"его","её","его","их")] артерии с помощью [src.declent_ru(GENITIVE)]! Кажется, [genderize_ru(user.gender,"он","она","оно","они")] пыта[pluralize_ru(user.gender,"ется","ются")] совершить самоубийство!</span>")
+	user.visible_message(span_suicide("[user] перереза[pluralize_ru(user.gender, "ет", "ют")] свои артерии с помощью [declent_ru(GENITIVE)]! Кажется, [genderize_ru(user.gender, "он", "она", "оно", "они")] пыта[pluralize_ru(user.gender, "ет", "ют")]ся совершить самоубийство!"))
```
Кажется, что

PlayerUnknown14 commented on code/game/objects/items/tools/wirecutters.dm:
```diff
@@ -68,6 +83,14 @@
 /obj/item/wirecutters/abductor
 	name = "alien wirecutters"
 	desc = "Очень острые кусачки, изготовленные из серебристо-зеленого металла."
```
ё*

PlayerUnknown14 commented on code/game/objects/items/tools/wirecutters.dm:
```diff
@@ -78,12 +101,28 @@
 /obj/item/wirecutters/cyborg
 	name = "wirecutters"
-	desc = "Это перерезает провода."
+	desc = "Инструмент для перерезания различных материалов."
```
```suggestion
	desc = "Инструмент для перерезания различных материалов. Специальная версия для роботизированных систем."
```

PlayerUnknown14 commented on code/game/objects/items/tools/wirecutters.dm:
```diff
@@ -92,11 +131,10 @@
 
 /obj/item/wirecutters/power/suicide_act(mob/user)
-	user.visible_message("<span class='suicide'>[user] обертывает [src.declent_ru(NOMINATIVE)] вокруг [genderize_ru(user.gender,"его","её","его","их")] шеи. Кажется, [genderize_ru(user.gender,"он","она","оно","они")] пыта[pluralize_ru(user.gender,"ется","ются")] оторвать [genderize_ru(user.gender,"его","её","его","их")] голову!</span>")
+	user.visible_message(span_suicide("[user] обёртывает [declent_ru(NOMINATIVE)] вокруг своей шеи. Кажется, [genderize_ru(user.gender, "он", "она", "оно", "они")] пыта[pluralize_ru(user.gender, "ет", "ют")]ся оторвать свою голову!"))
```
```suggestion
	user.visible_message(span_suicide("[user] обёртыва[pluralize_ru(user.gender, "ет", "ют")] [declent_ru(ACCUSATIVE)] вокруг своей шеи. Кажется, что [genderize_ru(user.gender, "он", "она", "оно", "они")] пыта[pluralize_ru(user.gender, "ет", "ют")]ся отрубить себе голову!"))
```

PlayerUnknown14 commented on code/game/objects/items/tools/wirecutters.dm:
```diff
@@ -109,6 +147,6 @@
 	playsound(get_turf(user), 'sound/items/change_jaws.ogg', 50, 1)
 	var/obj/item/crowbar/power/pryjaws = new /obj/item/crowbar/power
-	to_chat(user, "<span class='notice'Ты присоеденяешь поддевающую головку к [src.declent_ru(DATIVE)].</span>")
+	to_chat(user, span_notice("Ты присоеденяешь поддевающую головку к [declent_ru(DATIVE)]."))
```
Вы*

PlayerUnknown14 commented on code/game/objects/items/tools/wrench.dm:
```diff
@@ -19,32 +27,55 @@
 
 /obj/item/wrench/suicide_act(mob/user)
-	user.visible_message("<span class='suicide'>[user] избивает себя до смерти [src.declent_ru(INSTRUMENTAL)]! Похоже, [genderize_ru(user.gender,"он","она","оно","они")] пыта[pluralize_ru(user.gender,"ется","ются")] совершить самоубийсто!</span>")
+	user.visible_message(span_suicide("[user] избивает себя до смерти с помощью [declent_ru(GENITIVE)]! Похоже, [genderize_ru(user.gender, "он", "она", "оно", "они")] пыта[pluralize_ru(user.gender, "ет", "ют")]ся совершить самоубийсто!"))
```
```suggestion
	user.visible_message(span_suicide("[user] избива[pluralize_ru(user.gender, "ет", "ют")] себя до смерти с помощью [declent_ru(GENITIVE)]! Похоже, что [genderize_ru(user.gender, "он", "она", "оно", "они")] пыта[pluralize_ru(user.gender, "ет", "ют")]ся совершить самоубийство!"))
```

PlayerUnknown14 commented on code/game/objects/items/tools/wrench.dm:
```diff
@@ -19,32 +27,55 @@
 
 /obj/item/wrench/brass
 	name = "brass wrench"
 	desc = "Латунный ключ. Он слегка теплый на ощупь."
```
ё*

PlayerUnknown14 commented on code/game/objects/items/tools/wrench.dm:
```diff
@@ -19,32 +27,55 @@
 	name = "automatic wrench"
 	desc = "Продвинутый роботизированный гаечный ключ. Можно найти у строительных киборгов."
+	ru_names = list(
+		NOMINATIVE = "автоматической ключ",
```
```suggestion
		NOMINATIVE = "автоматический ключ",
```

PlayerUnknown14 commented on code/game/objects/items/tools/wrench.dm:
```diff
@@ -19,32 +27,55 @@
+		GENITIVE = "автоматического ключа",
+		DATIVE = "автоматическому ключу",
+		ACCUSATIVE = "автоматический ключ",
+		INSTRUMENTAL = "автоматическим ключем",
```
ключОм здесь и ниже.

PlayerUnknown14 commented on code/game/objects/items/tools/wrench.dm:
```diff
@@ -56,6 +87,14 @@
+	ru_names = list(
+		NOMINATIVE = "ручная дрель",
+		GENITIVE = "ручной дрели",
+		DATIVE = "ручную дрель",
```
```suggestion
		DATIVE = "ручной дрели",
```

PlayerUnknown14 commented on code/game/objects/items/tools/wrench.dm:
```diff
@@ -66,31 +105,37 @@
 	playsound(get_turf(user),'sound/items/change_drill.ogg', 50, 1)
 	var/obj/item/wirecutters/power/s_drill = new /obj/item/screwdriver/power
-	to_chat(user, "<span class='notice'>Ты присоеденяешь головку отвертки к [src.declent_ru(GENITIVE)].</span>")
+	to_chat(user, span_notice("Ты присоеденяешь головку отвертки к [declent_ru(GENITIVE)]."))
```
Вы*

PlayerUnknown14 commented on code/game/objects/items/tools/wrench.dm:
```diff
@@ -66,31 +105,37 @@
 
 /obj/item/wrench/power/suicide_act(mob/user)
-	user.visible_message("<span class='suicide'>[user] нажимает [src.declent_ru(INSTRUMENTAL)] против [genderize_ru(user.gender,"его","её","его","их")] головы! Похоже, [genderize_ru(user.gender,"он","она","оно","они")] пыта[pluralize_ru(user.gender,"ется","ются")] совершить самоубийство!")
+	user.visible_message(span_suicide("[user] нажимает [declent_ru(INSTRUMENTAL)] против своей головы! Похоже, [genderize_ru(user.gender, "он", "она", "оно", "они")] пыта[pluralize_ru(user.gender, "ет", "ют")]ся совершить самоубийство!"))
```
```suggestion
	user.visible_message(span_suicide("[user] закручива[pluralize_ru(user.gender, "ет", "ют")] [declent_ru(ACCUSATIVE)] против своей головы! Похоже, что [genderize_ru(user.gender, "он", "она", "оно", "они")] пыта[pluralize_ru(user.gender, "ет", "ют")]ся совершить самоубийство!"))
```

PlayerUnknown14 commented on code/game/objects/items/tools/wrench.dm:
```diff
@@ -66,31 +105,37 @@
 /obj/item/wrench/medical
 	name = "medical wrench"
-	desc = "Медицинский ключ с общим (медицинским?) Использование. Можно найти в руке."
+	desc = "Медецинский гаечный ключ. Используется для закручивания или откручивания чего либо, а так же в медецине."
```
```suggestion
	desc = "Медецинский гаечный ключ. Используется для закручивания или откручивания чего либо. Не спрашивайте, что отличает эту модель от обычного ключа."
```

PlayerUnknown14 commented on code/game/objects/items/tools/wrench.dm:
```diff
@@ -66,31 +105,37 @@
 
 /obj/item/wrench/medical/suicide_act(mob/living/user)
-	user.visible_message("<span class='suicide'>[user] молится медецинскому ключу чтобы он взял [genderize_ru(user.gender,"его","её","его","их")] душу. Похоже, [genderize_ru(user.gender,"он","она","оно","они")] пыта[pluralize_ru(user.gender,"ется","ются")] совершить самоубийство!</span>")
+	user.visible_message(span_suicide("[user] молится медецинскому ключу чтобы он взял [genderize_ru(user.gender, "его", "её", "его", "их")] душу. Похоже, [genderize_ru(user.gender, "он", "она", "оно", "они")] пыта[pluralize_ru(user.gender, "ет", "ют")]ся совершить самоубийство!"))
```
```suggestion
	user.visible_message(span_suicide("[user] молится [declent_ru(DATIVE)], чтобы он взял [genderize_ru(user.gender, "его", "её", "его", "их")] душу. Похоже, что [genderize_ru(user.gender, "он", "она", "оно", "они")] пыта[pluralize_ru(user.gender, "ет", "ют")]ся совершить самоубийство!"))
```

PlayerUnknown14 reviewed (commented):
Заставляет задуматься...

Dimempers committed:
Upd + RCD, RCL and cable

Поправил косяки и ошибки, перевел заодно РЦД(УБС), РЦЛ(БУП) и провода.

Dimempers committed:
Merge branch 'Translation-of-engineering-tools' of https://github.com/Dimempers/Paradise into Translation-of-engineering-tools

pilygun reviewed (dismissed):
косяков не найдено

Antoonij commented on code/game/objects/items/weapons/RCD.dm:
```diff
@@ -129,8 +137,8 @@
-	. += "<span class='notice'>MATTER: [matter]/[max_matter] matter-units.</span>"
-	. += "<span class='notice'>MODE: [mode].</span>"
+	. += span_notice("Материя: [matter]/[max_matter] юнитов-материи.")
+	. += span_notice("Мод: [mode].")
```
```suggestion
	. += span_notice("Режим: [mode].")
```
не переведено

Antoonij commented on code/game/objects/items/weapons/RCD.dm:
```diff
@@ -238,7 +246,7 @@
 			return
 	playsound(src, 'sound/effects/pop.ogg', 50, 0)
-	to_chat(user, "<span class='notice'>You change [src]'s mode to '[choice]'.</span>")
+	to_chat(user, span_notice("You change [src]'s mode to '[choice]")) // Оставляю другим локализаторам на перевод
```
```suggestion
	to_chat(user, span_notice("You change [src]'s mode to '[choice]")) 
```
Комментарии на русском в коде не стоит использовать

Antoonij commented on code/game/objects/items/weapons/RCL.dm:
```diff
@@ -48,7 +56,7 @@
 	if(!I.use_tool(src, user, 0, volume = I.tool_volume))
 		return
-	to_chat(user, "<span class='notice'>You loosen the securing screws on the side, allowing you to lower the guiding edge and retrieve the wires.</span>")
+	to_chat(user, span_notice("Вы откручиваете сбоку винты панели, что позваляет вам отсоеденить её и снять провода."))
```
```suggestion
	to_chat(user, span_notice("Вы откручиваете сбоку винты панели, что позволяет вам отсоединить её и снять провода."))
```
позволяет, отсоединить

Antoonij commented on code/game/objects/items/tools/wrench.dm:
```diff
@@ -69,25 +109,33 @@
 /obj/item/wrench/medical
 	name = "medical wrench"
-	desc = "A medical wrench with common (medical?) uses. Can be found in your hand."
+	desc = "Медецинский гаечный ключ. Используется для закручивания или откручивания чего либо. Не спрашивайте, что отличает эту модель от обычного ключа."
+	ru_names = list(
+		NOMINATIVE = "медецинский ключ",
+		GENITIVE = "медецинского ключа",
+		DATIVE = "медецинскому ключу",
+		ACCUSATIVE = "медецинской ключ",
+		INSTRUMENTAL = "медецинским ключом",
+		PREPOSITIONAL = "медецинском ключе"
+	)
```
Медицинский, а не медецинский

Antoonij commented on code/modules/power/cable.dm:
```diff
@@ -28,7 +28,15 @@ By design, d1 is the smallest direction and d2 is the highest
 	var/datum/powernet/powernet
 	name = "power cable"
-	desc = "A flexible superconducting cable for heavy-duty power transfer"
+	desc = "Гибкий сверхпроводящий кабель для передачи электроэнергии в сложных условиях"
```
```suggestion
	desc = "Гибкий сверхпроводящий кабель для передачи электроэнергии в суровых условиях"
```
Дело вкуса, суровых получше звучит чем сложных.

Antoonij commented on code/game/objects/items/tools/wirecutters.dm:
```diff
@@ -106,6 +147,6 @@
 	playsound(get_turf(user), 'sound/items/change_jaws.ogg', 50, 1)
 	var/obj/item/crowbar/power/pryjaws = new /obj/item/crowbar/power
-	to_chat(user, "<span class='notice'>You attach the pry jaws to [src].</span>")
+	to_chat(user, span_notice("Вы присоеденяете поддевающую головку к [declent_ru(DATIVE)]."))
```
```suggestion
	to_chat(user, span_notice("Вы присоединяете поддевающую головку к [declent_ru(DATIVE)]."))
```

Antoonij commented on code/game/objects/items/tools/welder.dm:
```diff
@@ -205,7 +213,15 @@
 /obj/item/weldingtool/largetank
 	name = "industrial welding tool"
-	desc = "A slightly larger welder with a larger tank."
+	desc = "Сварочный аппарат, но с баком побольше."
```
```suggestion
	desc = "Сварочный аппарат с увеличенным объемом бака."
```

Antoonij commented on code/game/objects/items/tools/welder.dm:
```diff
@@ -124,13 +132,13 @@
 	else
 		if(!silent)
-			to_chat(user, "<span class='warning'>You need more welding fuel to complete this task!</span>")
+			balloon_alert(user, ("Тебе нужно больше сварочного топлива в [declent_ru(PREPOSITIONAL)]!"))
```
длинный баллун алерт, сократить до 3-4 слов

Antoonij commented on code/game/objects/items/tools/screwdriver.dm:
```diff
@@ -104,18 +144,18 @@
 	playsound(get_turf(user), 'sound/items/change_drill.ogg', 50, 1)
 	var/obj/item/wrench/power/b_drill = new /obj/item/wrench/power
-	to_chat(user, "<span class='notice'>You attach the bolt driver bit to [src].</span>")
+	to_chat(user, span_notice("Вы присоеденяете головку болтового сверла к [declent_ru(DATIVE)]."))
```
```suggestion
	to_chat(user, span_notice("Вы присоединяете головку болтового сверла к [declent_ru(DATIVE)]."))
```

Antoonij commented on code/game/objects/items/tools/multitool.dm:
```diff
@@ -111,7 +119,7 @@
 
 /obj/item/multitool/ai_detect/admin
-	desc = "Used for pulsing wires to test which to cut. Not recommended by doctors. Has a strange tag that says 'Grief in Safety'" //What else should I say for a meme item?
+	desc = "Используется для пульсации проводов, чтобы проверить, какой из них следует разрезать. Не рекомендуется врачами. Имеет странную бирку с надписью  'Гриферить в удовольствие!'" //What else should I say for a meme item?
```
```suggestion
	desc = "Используется для пульсации проводов, чтобы проверить, какой из них следует разрезать. Не рекомендуется врачами. Имеет странную бирку с надписью  'Гриферить в удовольствие!'" // What else should I say for a meme item?
```

Antoonij commented on code/game/objects/items/tools/holotool.dm:
```diff
@@ -6,15 +6,22 @@ Holotool. All instruments in one object
 /obj/item/holotool
 	name = "experimental holotool"
-	desc = "A highly experimental holographic tool projector."
+	desc = "Эксперементальный голографический инструмент-проектор."
+	ru_names = list(
+		NOMINATIVE = "эксперементальный голотул",
+		GENITIVE = "эксперементального голотула",
+		DATIVE = "эксперементальному голотулу",
+		ACCUSATIVE = "эксперементальный голотул",
+		INSTRUMENTAL = "эксперементальным голотулом",
+		PREPOSITIONAL = "эксперементальном голотуле"
+	)
```
эксперИментальный, рунеймы также поменять

Antoonij commented on code/game/objects/items/tools/crowbar.dm:
```diff
@@ -83,21 +130,20 @@
 	playsound(get_turf(user), 'sound/items/change_jaws.ogg', 50, 1)
 	var/obj/item/wirecutters/power/cutjaws = new /obj/item/wirecutters/power
-	to_chat(user, "<span class='notice'>You attach the cutting jaws to [src].</span>")
+	to_chat(user, span_notice("Вы присоеденяете режущую головку к [declent_ru(DATIVE)]."))
```
```suggestion
	to_chat(user, span_notice("Вы присоединяете режущую головку к [declent_ru(DATIVE)]."))
```

Dimempers committed:
minor upd

Dimempers committed:
Minor upd x2

Dimempers committed:
Minor upd x3

Поменял в нект. балунах деклент_ру на обобщённое название для сокращения первых.

Dimempers committed:
Minor upd x4

В прошлый раз забыл сохранить это, только щас увидел что не сохранилось, исправляю. Так же кое-где поправил ошибки.

Dimempers committed:
Ultra minor upd

+1-1 Буковка ё в 1 месте.

PlayerUnknown14 commented on code/game/objects/items/tools/multitool.dm:
```diff
@@ -49,12 +49,12 @@
 		return
 	buffer = M
-	to_chat(user, span_notice("Ты загрузил [M] во внутренний буфер [declent_ru(GENITIVE)]."))
+	to_chat(user, span_notice("Вы загрузили [M] во внутренний буфер [declent_ru(GENITIVE)]."))
```
```suggestion
	to_chat(user, span_notice("Вы загрузили [M.declent_ru(ACCUSATIVE)] во внутренний буфер [declent_ru(GENITIVE)]."))
```

PlayerUnknown14 commented on code/game/objects/items/tools/welder.dm:
```diff
@@ -96,14 +96,14 @@
 /obj/item/weldingtool/attack_self(mob/user)
 	if(tool_enabled) //Turn off the welder if it's on
-		to_chat(user, span_notice("Вы выключили [declent_ru(NOMINATIVE)]."))
+		balloon_alert(user, ("Вы выключили сварочный аппарат."))
```
```suggestion
		balloon_alert(user, ("выключено"))
```

PlayerUnknown14 commented on code/game/objects/items/tools/welder.dm:
```diff
@@ -70,7 +70,7 @@
 /obj/item/weldingtool/can_enter_storage(obj/item/storage/S, mob/user)
 	if(tool_enabled)
-		to_chat(user, span_warning("[capitalize(declent_ru(NOMINATIVE))] невозможно положить пока он включен!"))
+		to_chat(user, span_warning("[capitalize(declent_ru(ACCUSATIVE))] невозможно положить, пока он включён!"))
```
```suggestion
		balloon_alert(user, "включено!")
```

PlayerUnknown14 commented on code/game/objects/items/tools/welder.dm:
```diff
@@ -96,14 +96,14 @@
 		return
 	else if(GET_FUEL) //The welder is off, but we need to check if there is fuel in the tank
-		to_chat(user, span_notice("Вы включили [declent_ru(NOMINATIVE)]."))
+		balloon_alert(user, ("Вы включили сварочный аппарат."))
```
```suggestion
		balloon_alert(user, "выключено")
```

PlayerUnknown14 commented on code/game/objects/items/tools/welder.dm:
```diff
@@ -96,14 +96,14 @@
 		toggle_welder()
 	else //The welder is off and unfuelled
-		to_chat(user, span_notice("В [declent_ru(PREPOSITIONAL)] закончилось топливо!"))
+		balloon_alert(user, ("В сварочном аппарате закончилось топливо!"))
```
```suggestion
		balloon_alert(user, "закончилось топливо!")
```

PlayerUnknown14 commented on code/game/objects/items/tools/welder.dm:
```diff
@@ -132,13 +132,13 @@
 	else
 		if(!silent)
-			to_chat(user, span_warning("Тебе нужно больше сварочного топлива в [declent_ru(PREPOSITIONAL)] для выполнения этой задачи!"))
+			balloon_alert(user, ("Недостаточно топлива!"))
```
```suggestion
			balloon_alert(user, "недостаточно топлива!")
```

PlayerUnknown14 commented on code/game/objects/items/tools/welder.dm:
```diff
@@ -132,13 +132,13 @@
 	if(!tool_enabled)
 		if(!silent)
-			to_chat(user, span_notice("[capitalize(declent_ru(NOMINATIVE))] должен быть включён для выполнения этой задачи!"))
+			balloon_alert(user, ("сварочный аппарат должен быть включён!"))
```
```suggestion
			balloon_alert(user, "выключено!")
```

PlayerUnknown14 commented on code/game/objects/items/tools/welder.dm:
```diff
@@ -179,16 +179,16 @@
 		return
 	if(GET_FUEL >= maximum_fuel)
-		to_chat(user, span_notice("[capitalize(declent_ru(NOMINATIVE))] уже полон!"))
+		balloon_alert(user, ("сварочный аппарат уже полон!"))
```
```suggestion
		balloon_alert(user, "топливный бак полон!")
```

PlayerUnknown14 commented on code/game/objects/items/tools/welder.dm:
```diff
@@ -179,16 +179,16 @@
 	var/amount_transferred = A.reagents.trans_id_to(src, "fuel", amount)
 	if(amount_transferred)
-		to_chat(user, span_notice("Ты заполнил [declent_ru(NOMINATIVE)] [amount_transferred] u.</span>"))
+		balloon_alert(user, ("Ты заполнил сварочный аппарат [amount_transferred] u топлива."))
```
```suggestion
		balloon_alert(user, "заправлено")
```

PlayerUnknown14 commented on code/game/objects/items/tools/welder.dm:
```diff
@@ -179,16 +179,16 @@
 		return amount_transferred
 	else
-		balloon_alert(user, span_warning("В [A] недостатачно топлива!</span>"))
+		balloon_alert(user, ("В [A] недостатачно топлива!"))
```
```suggestion
		balloon_alert(user, "недостаточно топлива!")
```

PlayerUnknown14 commented on code/game/objects/items/tools/welder.dm:
```diff
@@ -213,7 +213,7 @@
 /obj/item/weldingtool/largetank
 	name = "industrial welding tool"
-	desc = "Сварочный аппарат, но с баком побольше."
+	desc = "Сварочный аппарат с увеличенным объемом бака."
```
```suggestion
	desc = "Сварочный аппарат с баком побольше."
```

PlayerUnknown14 commented on code/game/objects/items/tools/wirecutters.dm:
```diff
@@ -49,7 +49,7 @@
 	if(istype(target) && istype(target.handcuffed, /obj/item/restraints/handcuffs/cable))
 		var/obj/item/cuffs = target.handcuffed
 		user.visible_message(
```
```suggestion
		visible_message("[user] перерезае[pluralize_ru(user.gender, "ет", "ют")] стяжки [target], используя [declent_ru(ACCUSATIVE)]!")
```

PlayerUnknown14 commented on code/game/objects/items/tools/wirecutters.dm:
```diff
@@ -49,7 +49,7 @@
-			span_notice("[user] перерезает стяжки [target]'s с помощью [declent_ru(GENITIVE)]!"),
+			balloon_alert("стяжки перерезаны!"),
 			span_notice("Ты перерезал стяжки [target]'s с помощью [declent_ru(GENITIVE)]!"),
 		)
```
```suggestion
```

PlayerUnknown14 commented on code/game/objects/items/tools/wirecutters.dm:
```diff
@@ -49,7 +49,7 @@
 		var/obj/item/cuffs = target.handcuffed
 		user.visible_message(
-			span_notice("[user] перерезает стяжки [target]'s с помощью [declent_ru(GENITIVE)]!"),
+			balloon_alert("стяжки перерезаны!"),
```
```suggestion
			balloon_alert(user, "стяжки перерезаны")
```

PlayerUnknown14 commented on code/game/objects/items/tools/wirecutters.dm:
```diff
@@ -147,6 +147,6 @@
 	playsound(get_turf(user), 'sound/items/change_jaws.ogg', 50, 1)
 	var/obj/item/crowbar/power/pryjaws = new /obj/item/crowbar/power
-	to_chat(user, span_notice("Ты присоеденяешь поддевающую головку к [declent_ru(DATIVE)]."))
+	to_chat(user, span_notice("Вы присоединяете поддевающую головку к [declent_ru(DATIVE)]."))
```
```suggestion
	balloon_alert(user, "режим монтировки")
```

PlayerUnknown14 commented on code/game/objects/items/tools/wrench.dm:
```diff
@@ -109,24 +109,24 @@
 	playsound(get_turf(user),'sound/items/change_drill.ogg', 50, 1)
 	var/obj/item/wirecutters/power/s_drill = new /obj/item/screwdriver/power
-	to_chat(user, span_notice("Ты присоеденяешь головку отвертки к [declent_ru(GENITIVE)]."))
+	to_chat(user, span_notice("Вы присоеденяете головку отвёртки к [declent_ru(DATIVE)]."))
```
```suggestion
	balloon_alert(user, "режим отвёртки")
```

PlayerUnknown14 commented on code/game/objects/items/tools/crowbar.dm:
```diff
@@ -137,13 +137,13 @@
 	playsound(get_turf(user), 'sound/items/change_jaws.ogg', 50, 1)
 	var/obj/item/wirecutters/power/cutjaws = new /obj/item/wirecutters/power
-	to_chat(user, span_notice("Вы присоеденяете режущую головку к [declent_ru(DATIVE)]."))
+	to_chat(user, span_notice("Вы присоединяете режущую головку к [declent_ru(DATIVE)]."))
```
```suggestion
	balloon_alert(user, "режим кусачек")
```

PlayerUnknown14 commented on code/game/objects/items/tools/screwdriver.dm:
```diff
@@ -136,18 +144,18 @@
 	playsound(get_turf(user), 'sound/items/change_drill.ogg', 50, 1)
 	var/obj/item/wrench/power/b_drill = new /obj/item/wrench/power
-	to_chat(user, span_notice("Вы присоеденяете головку болтового сверла к [declent_ru(GENITIVE)]."))
+	to_chat(user, span_notice("Вы присоединяете головку болтового сверла к [declent_ru(DATIVE)]."))
```
```suggestion
	balloon_alert(user, "режим гаечного ключа")
```

PlayerUnknown14 commented on code/game/objects/items/tools/wrench.dm:
```diff
@@ -135,7 +135,7 @@
 
 /obj/item/wrench/medical/suicide_act(mob/living/user)
-	user.visible_message(span_suicide("[user] молится медецинскому ключу чтобы он взял [genderize_ru(user.gender, "его", "её", "его", "их")] душу. Похоже, [genderize_ru(user.gender, "он", "она", "оно", "они")] пыта[pluralize_ru(user.gender, "ет", "ют")]ся совершить самоубийство!"))
+	user.visible_message(span_suicide("[user] молится [declent_ru(DATIVE)] чтобы он взял [genderize_ru(user.gender, "его", "её", "его", "их")] душу. Похоже, что [genderize_ru(user.gender, "он", "она", "оно", "они")] пыта[pluralize_ru(user.gender, "ет", "ют")]ся совершить самоубийство!"))
```
```suggestion
	user.visible_message(span_suicide("[user] мол[pluralize_ru(user.gender, "ит", "ят")]ся [declent_ru(DATIVE)], чтобы он взял [genderize_ru(user.gender, "его", "её", "его", "их")] душу. Похоже, что [genderize_ru(user.gender, "он", "она", "оно", "они")] пыта[pluralize_ru(user.gender, "ет", "ют")]ся совершить самоубийство!"))
```

PlayerUnknown14 commented on code/game/objects/items/weapons/RCD.dm:
```diff
@@ -129,8 +137,8 @@
 	. = ..()
-	. += "<span class='notice'>MATTER: [matter]/[max_matter] matter-units.</span>"
-	. += "<span class='notice'>MODE: [mode].</span>"
+	. += span_notice("Материя: [matter]/[max_matter] юнитов-материи.")
```
```suggestion
	. += span_notice("Материя: [matter]/[max_matter] единиц.")
```

PlayerUnknown14 commented on code/game/objects/items/weapons/RCD.dm:
```diff
@@ -178,21 +186,21 @@
 /obj/item/rcd/proc/rcd_reload(obj/item/rcd_ammo/rcd_ammo, mob/user)
 	if(matter >= max_matter)
-		to_chat(user, "<span class='notice'>The RCD can't hold any more matter-units.</span>")
+		to_chat(user, span_notice("УБС не может принять больше юнитов-материи."))
```
```suggestion
		balloon_alert(user, "нет места!")
```

PlayerUnknown14 commented on code/game/objects/items/weapons/RCD.dm:
```diff
@@ -178,21 +186,21 @@
 
 	if(!user.drop_item_ground(rcd_ammo))
-		to_chat(user, "<span class='warning'>[rcd_ammo] is stuck to your hand!</span>")
+		to_chat(user, span_warning("[rcd_ammo] is stuck to your hand!"))
```
```suggestion
		balloon_alert(user, "невозможно отцепить!")
```

PlayerUnknown14 commented on code/game/objects/items/weapons/RCD.dm:
```diff
@@ -178,21 +186,21 @@
 		qdel(rcd_ammo)
 		playsound(loc, 'sound/machines/click.ogg', 50, 1)
-		to_chat(user, "<span class='notice'>The RCD now holds [matter]/[max_matter] matter-units.</span>")
+		to_chat(user, span_notice("Теперь в УБС [matter]/[max_matter] юнитов-материи."))
```
```suggestion
		balloon_alert(user, "картридж вставлен")
```

PlayerUnknown14 commented on code/game/objects/items/weapons/RCD.dm:
```diff
@@ -178,21 +186,21 @@
+		to_chat(user, span_notice("Теперь в УБС [matter]/[max_matter] юнитов-материи."))
 	else
-		to_chat(user, "<span class='warning'>This matter cartridge is incompatible with your RCD</span>")
+		to_chat(user, span_warning("Этот картридж материи несовместим с вашим УБС"))
```
```suggestion
		balloon_alert(user, "несовместимо!")
```

PlayerUnknown14 commented on code/game/objects/items/weapons/RCD.dm:
```diff
@@ -238,7 +246,7 @@
 			return
 	playsound(src, 'sound/effects/pop.ogg', 50, 0)
-	to_chat(user, "<span class='notice'>You change [src]'s mode to '[choice]'.</span>")
+	to_chat(user, span_notice("You change [src]'s mode to '[choice]"))
```
```suggestion
	to_chat(user, span_notice("Вы меняете режим [declent_ru(GENITIVE)] на [choice]."))
	balloon_alert(user, "режим изменён")
```

PlayerUnknown14 commented on code/game/objects/items/weapons/RCD.dm:
```diff
@@ -376,7 +384,7 @@
 	var/area/check_area = get_area(target)
 	if(check_area?.type in areas_blacklist)
-		to_chat(user, span_warning("Something prevents you from using [src] in here..."))
+		to_chat(user, span_warning("Чтото мешает вам использовать [declent_ru(NOMINATIVE)] здесь..."))
```
```suggestion
		to_chat(user, span_warning("Что-то мешает вам использовать [declent_ru(ACCUSATIVE)] здесь."))
		balloon_alert(user, "невозможно!")
```

PlayerUnknown14 commented on code/game/objects/items/weapons/RCD.dm:
```diff
@@ -434,7 +442,7 @@
 	if(is_taipan(z) || is_admin_level(z)) //Защищает тайпан и админские Z-lvla от взрыва RCD
 		return
-	audible_message("<span class='danger'><b>[src] begins to vibrate and buzz loudly!</b></span>", "<span class='danger'><b>[src] begins vibrating violently!</b></span>")
+	audible_message(span_danger("<b>[capitalize(declent_ru(ACCUSATIVE))] начинает громко вибрировать и жужжать!</b>"), span_danger("<b>[capitalize(declent_ru(ACCUSATIVE))] начинает сильно вибрировать!</b>"))
```
```suggestion
	audible_message(span_danger("<b>[capitalize(declent_ru(NOMINATIVE))] начинает громко вибрировать и жужжать!</b>"), span_danger("<b>[capitalize(declent_ru(NOMINATIVE))] начинает сильно вибрировать!</b>"))
```

PlayerUnknown14 commented on code/game/objects/items/weapons/RCD.dm:
```diff
@@ -458,7 +466,15 @@
 /obj/item/rcd_ammo
 	name = "compressed matter cartridge"
-	desc = "Highly compressed matter for the RCD."
+	desc = "Сильно спресованная материя в картриджи для УБС."
```
```suggestion
	desc = "Картридж для УБС, содержащий в себе спрессованную материю."
```

PlayerUnknown14 commented on code/game/objects/items/weapons/RCD.dm:
```diff
@@ -458,7 +466,15 @@
-	desc = "Highly compressed matter for the RCD."
+	desc = "Сильно спресованная материя в картриджи для УБС."
+	ru_names = list(
+		NOMINATIVE = "картридж материи",
+		GENITIVE = "картриджа материи",
+		DATIVE = "картриджу материи",
+		ACCUSATIVE = "картридж материи",
+		INSTRUMENTAL = "картриджом материи",
+		PREPOSITIONAL = "картридже материи"
```
```suggestion
		NOMINATIVE = "картридж УБС",
		GENITIVE = "картриджа УБС",
		DATIVE = "картриджу УБС",
		ACCUSATIVE = "картридж УБС",
		INSTRUMENTAL = "картриджем УБС",
		PREPOSITIONAL = "картридже УБС"
```

PlayerUnknown14 commented on code/game/objects/items/weapons/RCL.dm:
```diff
@@ -26,17 +34,17 @@
 			loaded.max_amount = max_amount //We store a lot.
 			update_icon(UPDATE_ICON_STATE)
-			to_chat(user, span_notice("You add the cables to the [src]. It now contains [loaded.amount]."))
+			to_chat(user, span_notice("Вы загрузили провода в [declent_ru(ACCUSATIVE)]. В хранилище теперь [loaded.amount]."))
```
```suggestion
			balloon_alert(user, "провода загружены")
```

PlayerUnknown14 commented on code/game/objects/items/weapons/RCL.dm:
```diff
@@ -26,17 +34,17 @@
 			return ATTACK_CHAIN_BLOCKED_ALL
 		if(loaded.amount >= max_amount)
-			to_chat(user, span_warning("The [name]'s cable storage is full."))
+			balloon_alert(user, ("Хранилище проводов в [declent_ru(PREPOSITIONAL)] заполнено."))
```
```suggestion
			balloon_alert(user, "нет места!")
```

PlayerUnknown14 commented on code/game/objects/items/weapons/RCL.dm:
```diff
@@ -26,17 +34,17 @@
+			balloon_alert(user, ("Хранилище проводов в [declent_ru(PREPOSITIONAL)] заполнено."))
 			return ATTACK_CHAIN_PROCEED
-		to_chat(user, span_notice("You load some cable into [src]."))
+		balloon_alert(user, ("Вы загрузили провода в [declent_ru(ACCUSATIVE)]."))
```
```suggestion
		balloon_alert(user, "провода загружены")
```

PlayerUnknown14 commented on code/game/objects/items/weapons/RCL.dm:
```diff
@@ -26,17 +34,17 @@
 		loaded.amount = amount
 		update_icon(UPDATE_ICON_STATE)
-		to_chat(user, span_notice("You add the cables to the [src]. It now contains [loaded.amount]."))
+		to_chat(user, span_notice("Вы загрузили провода в [declent_ru(ACCUSATIVE)]. В хранилище теперь [loaded.amount]."))
```
```suggestion
		balloon_alert(user, "провода загружены")
```

PlayerUnknown14 commented on code/game/objects/items/weapons/RCL.dm:
```diff
@@ -48,7 +56,7 @@
 	if(!I.use_tool(src, user, 0, volume = I.tool_volume))
 		return
-	to_chat(user, "<span class='notice'>You loosen the securing screws on the side, allowing you to lower the guiding edge and retrieve the wires.</span>")
+	to_chat(user, span_notice("Вы откручиваете сбоку винты панели, что позволяет вам отсоединить её и снять провода."))
```
```suggestion
	balloon_alert("боковые винты откручены")
```

PlayerUnknown14 commented on code/game/objects/items/weapons/RCL.dm:
```diff
@@ -99,7 +107,7 @@
 	if(!loaded || !loaded.amount)
 		if(loud)
-			to_chat(user, "<span class='notice'>The last of the cables unreel from [src].</span>")
+			to_chat(user, span_notice("Последний провод был размотан с [declent_ru(GENITIVE)]."))
```
```suggestion
			balloon_alert(user, "провода закончились!")
```

PlayerUnknown14 commented on code/game/objects/items/weapons/RCL.dm:
```diff
@@ -135,7 +143,7 @@
 /obj/item/twohanded/rcl/proc/trigger(mob/user)
 	if(is_empty(user, 0))
-		to_chat(user, "<span class='warning'>\The [src] is empty!</span>")
+		balloon_alert(user, ("[declent_ru(ACCUSATIVE)] пуст!"))
```
```suggestion
		balloon_alert(user, "провода отсутствуют!")
```

PlayerUnknown14 commented on code/modules/power/cable.dm:
```diff
@@ -28,7 +28,15 @@ By design, d1 is the smallest direction and d2 is the highest
 	var/datum/powernet/powernet
 	name = "power cable"
-	desc = "A flexible superconducting cable for heavy-duty power transfer"
+	desc = "Гибкий сверхпроводящий кабель для передачи электроэнергии в суровых условиях"
```
```suggestion
	desc = "Гибкий сверхпроводящий кабель для передачи электроэнергии в самых суровых условиях"
```

PlayerUnknown14 commented on code/modules/power/cable.dm:
```diff
@@ -166,14 +174,14 @@ By design, d1 is the smallest direction and d2 is the highest
 
 	if((our_turf.transparent_floor == TURF_TRANSPARENT) || our_turf.intact)
-		to_chat(user, span_danger("You cannot interact with something that's under the floor!"))
+		to_chat(user, span_danger("Вы не можете взаимодействовать с тем, что находится под полом!"))
```
```suggestion
		balloon_alert(user, "невозможно!")
```

PlayerUnknown14 commented on code/modules/power/cable.dm:
```diff
@@ -166,14 +174,14 @@ By design, d1 is the smallest direction and d2 is the highest
 		var/obj/item/stack/cable_coil/coil = I
 		if(coil.get_amount() < 1)
-			to_chat(user, span_warning("Not enough cable!"))
+			balloon_alert(user, ("Не хватает провода!"))
```
```suggestion
			balloon_alert(user, "недостаточно проводов!")
```

PlayerUnknown14 commented on code/modules/power/cable.dm:
```diff
@@ -182,7 +190,7 @@ By design, d1 is the smallest direction and d2 is the highest
 		var/obj/item/twohanded/rcl/rcl = I
 		if(!rcl.loaded)
-			to_chat(user, span_warning("The [rcl.name] has no cable!"))
+			to_chat(user, span_warning("В [rcl.declent_ru(PREPOSITIONAL)] нет проводов!"))
```
```suggestion
			balloon_alert(user, "недостаточно проводов!")
```

PlayerUnknown14 commented on code/modules/power/cable.dm:
```diff
@@ -209,22 +217,22 @@ By design, d1 is the smallest direction and d2 is the highest
 	var/turf/T = get_turf(src)
 	if((T.transparent_floor == TURF_TRANSPARENT) || T.intact)
-		to_chat(user, "<span class='danger'>You can't interact with something that's under the floor!</span>")
+		to_chat(user, span_danger("Вы не можете взаимодействовать с тем, что находится под полом!"))
```
```suggestion
		balloon_alert(user, "невозможно!")
```

PlayerUnknown14 commented on code/modules/power/cable.dm:
```diff
@@ -209,22 +217,22 @@ By design, d1 is the smallest direction and d2 is the highest
 	if(shock(user, 50))
 		return
-	user.visible_message("[user] cuts the cable.", "<span class='notice'>You cut the cable.</span>")
+	user.visible_message("[user] обрезал провода.", span_notice("Вы обрезали провода."))
```
```suggestion
	visible_message("[user] срезал[genderize_ru(user.gender, "", "а", "о", "и")] [declent_ru(ACCUSATIVE)].")
	balloon_alert(user, "провод срезан!")
```

PlayerUnknown14 reviewed (commented):
Далеко от идеала.

PlayerUnknown14 committed:
Merge branch 'master220' into Translation-of-engineering-tools

PlayerUnknown14 committed:
Apply suggestions from code review

PlayerUnknown14 commented:
Ой бля.
